site/src/planck-io.md

```diff
@@ -171,7 +186,25 @@ Returns a map containing the attributes of the item at a given `path`.
 Spec
  _args_: `(cat :path (nillable? (or :string string? :file file?)))`<br/>
  _ret_: `map?`
- 
+  
+### <a name="file-name"></a>file-name
+`([x])`
+
+Returns the name (the final path element) of `x`.
+
+Spec<br/>
+ _args_: `(cat :x (or :string string? :file file?))`<br/>
+ _ret_: `(s/nilable string?)`<br/>
+
+### <a name="hidden-file?"></a>hidden-file?
+`([x])`
+
+Checks if `x` is hidden (name begins with a '.' character).  Note that this fn does not take MS-DOS' "hidden attribute" into account.
```

Comment:
The copy regarding `MS-DOS` doesn't match the docstring. (Perhaps this should be removed; Planck doesn't run under MS-DOS.)

Reply:
This also applies to Windows, since it inherited the MS-DOS filesystem.  I think it’s worth calling out, since Java’s implementation takes the MS-DOS hidden attribute into account (on that platform only, of course).  

Though if Planck doesn’t run on Windows either it’s a bit moot. Does it run on Windows?

Reply:
Planck does not run on Windows. The current targets are macOS and Linux.

Reply:
Fixed in rev cf00676.